### ISSUE TITLE
Making dropdown menus keyboard accessible

### ISSUE THREAD

**Issue by pilatesqueen** [wp-bootstrap/wp-bootstrap-navwalker#392]
Hi,
I've been using the nav walker on my projects with Sage started theme. In one of my current projects I need to make the dropdown menus keyboard accessible with minimal JS.

Thanks

Has anyone been able to accomplish this?

#### Steps to reproduce the issue:
The menu itself can be navigated via keyboard but the dropdown items get ignored.

**Comment by pattonwebz** [comment]
Of course it makes perfect sense for the menus to be fully keyboard accessible. And it really should be because I had some people from WP accessibility team test an active install of it not so long ago and they didn't flag this. Perhaps I have a misplaced aria value or something to that effect between then and now.

You should be able to toggle open/close dropdowns by pressing `return` on them. Does the menu not do that for you?

**Comment by pilatesqueen** [comment]
The keyboard navigation works only if the parent menu item is not a link (the walker automatically replaces the parent item with a blank href attribute.) 
After disabling the blank href the parent menu item works (is a clickable nav link) but the dropdowns can't be accessed via keyboard. To me it seems that keeping parent item non-clickable is confusing to the users

**Comment by pattonwebz** [comment]
For me the keyboard navigation is working fine, I do not believe I have changed anything between when you opened this issue and now that would have resolved any problems but have a look at the gif and let me know if that is not the expected way it should work.

![Peek 2019-10-02 19-48](https://user-images.githubusercontent.com/3902039/66072793-a41b3600-e54d-11e9-8aa5-b97595cbcd53.gif)

**Comment by Sakib3s** [comment]
@pattonwebz Your provided example not working with Bootstrap v4.5.2. Would you please check with  Bootstrap v4.5.2?

Regards

**Comment by IanDelMar** [comment]
@Sakib3s It's working for me (Bootstrap 4.6.0) in IE 11, Edge 88, Chrome 87. I doubt that this is different with v4.5.2.

**Comment by Sakib3s** [comment]
@IanDelMar Thanks for the massage! I have fixed it with CSS.

**Comment by IanDelMar** [comment]
Closing this because the issue was not reproduceable, there was no feedback from the OP and using a "blank" (actually, it's not blank but `href="#"`')  href is valid Bootstrap markup.
